"Clr -msbuild" build is broken by clrinterpreter

```
.\build.cmd Clr -c Checked -msbuild
```
fails with:
```
  CMake Error in CMakeLists.txt:
    The EXCLUDE_FROM_ALL property of target "clrinterpreter" varies by
    configuration.  This is not supported by the "Visual Studio 17 2022"
    generator.


  -- Generating done (2.6s)
  CMake Generate step failed.  Build files cannot be regenerated correctly.
  BUILD: Error: failed to generate native component build project
```

--- a/src/coreclr/interpreter/CMakeLists.txt
+++ b/src/coreclr/interpreter/CMakeLists.txt
@@ -49,7 +49,9 @@ endif()
 
 add_library_clr(clrinterpreter ${LIBRARY_TYPE} ${INTERPRETER_SOURCES})
 
-set_target_properties(clrinterpreter PROPERTIES EXCLUDE_FROM_ALL $<NOT:${FEATURE_INTERPRETER}>)
+if (NOT CMAKE_GENERATOR MATCHES "Visual Studio")
+  set_target_properties(clrinterpreter PROPERTIES EXCLUDE_FROM_ALL $<NOT:${FEATURE_INTERPRETER}>)
+endif()
 
 add_dependencies(clrinterpreter interpreter_exports)
 

